browser/app/js/objects/ObjectsBulkActions.js

```diff
@@ -29,10 +42,14 @@ export class ObjectsBulkActions extends React.Component {
     }
   }
   handleDownload() {
-    const { checkedObjects, clearChecked, downloadChecked, downloadObject } = this.props
-    if (checkedObjects.length === 1 && !checkedObjects[0].endsWith('/')) {
+    const {
+      checkedObjects,
+      clearChecked,
+      downloadChecked,
+      downloadObject
+    } = this.props
+    if (checkedObjects.length === 1 && !checkedObjects[0].endsWith("/")) {
       downloadObject(checkedObjects[0])
-      clearChecked()
```

Comment:
Do we want to not clear checked objects on download? If this is the case we can keep this change as is.

Reply:
I think we should not clear the checked objects as there are some useful use cases with it. For an example let's say, we have hand picked few objects out of a big list and we need to remove them after downloading. If we call the clear function after download, we may need to re-pick all the objects again in order to delete them. I've also checked with some of the popular players such as Dropbox and they are all doing the same.